docs/release-notes.md

```diff
@@ -3,6 +3,43 @@ nav_order: 9
 ---
 
 # Release notes
+## Butane 0.15.0 (2022-06-21)
+
+### Features
+
+- Add arm64 support for butane container
+- Add Flatcar support
+- Stabilize OpenShift spec 4.10
+- Add GRUB password support
+- Add TranslationSet.Map() to the Go API
+- Add AddFromCommonObject() to the Go API
+
+### Bug fixes
+
+- Fix build on Windows
+- Fix Windows path handling
+
+### Misc. changes
+
+- Improve cross-platform consistency 
+- Update Dockerfile.validate to install less dependencies
+- Add error for Clevis support
+- Correctly abort on a failed specfile fetch during `make`
+- Switch to full base image for script support in container
```

Comment:
Hmm.  I put the extra words in "support use in multi-stage builds" because I'm hoping to make it as clear as possible that this is a user-facing feature rather than an internal change.